### DISCUSSION TITLE
GC9A01(1.09) 240X240)  with 8080 8 BIT on ESP32

### DISCUSSION THREAD

**Discussion by Black-Blade** [Bodmer/TFT_eSPI#1710] [New processor/display support or capability request]
Hello everybody
can't get the display to work.

What am I doing wrong

Here my User_Setup.h 

`#define ESP32_PARALLEL
#define GC9A01_DRIVER
#define TFT_WIDTH  240
#define TFT_HEIGHT 240
// ESP32 pins used for UNO format board
#define TFT_CS   33  
#define TFT_DC   15  
#define TFT_RST  32  

#define TFT_WR    4  
#define TFT_RD    2

#define TFT_D0   12  
#define TFT_D1   13  
#define TFT_D2   26
#define TFT_D3   25
#define TFT_D4   17
#define TFT_D5   16
#define TFT_D6   27
#define TFT_D7   14
`
Thank you very much

**Comment by SpaceTeddy** [comment]
have exactly the same LCD here. any success`?

**Reply by FarazFKhan** [reply]
Check out the following video on how to use this display with the TPT_eSPI library:

https://www.youtube.com/watch?v=k2c2zCmC_X0&t=2107s
